packages/apputils-extension/src/notificationplugin.tsx

```diff
@@ -678,15 +678,16 @@ namespace Private {
 
   export function CloseButton(props: ICloseButtonProps) {
     return (
-      <button
-        className={`jp-Button jp-mod-minimal ${TOAST_CLOSE_BUTTON_CLASS}${
+      <Button
+        className={`jp-Button jp-icon-hover ${TOAST_CLOSE_BUTTON_CLASS}${
```

Comment:
Let's improve the style by using the appearance `stealth`:

```suggestion
        appearance="stealth"
        className={`jp-Button jp-icon-hover ${TOAST_CLOSE_BUTTON_CLASS}${
```